File: 프로그래머스/Lv.0/181922. 수열과 구간 쿼리 4/수열과 구간 쿼리 4.java
class Solution {
    public int[] solution(int[] arr, int[][] queries) {
        int[] answer = new int[queries.length];
        for(int i = 0; i < queries.length; i++){
            int[] temp = queries[i];
             for(int j = temp[0]; j <= temp[1]; j++){
                if(j % temp[2] == 0){
                    arr[j] += 1;
                }
             }
        }
        answer = arr;
        return answer;
    }
}

File: 프로그래머스/Lv.0/181922. 수열과 구간 쿼리 4/수열과 구간 쿼리 4.py
def solution(arr, queries):
    answer = []
    for s, e, k in queries:
        for i in range(s,e+1):
            if not k:
                arr[i] = arr[i]+1
                continue
                
            if i%k ==0:
                arr[i] = arr[i]+1
    answer = arr
    return answer